tamil/README.md

```diff
@@ -0,0 +1,6 @@
+These words were taken from Tamil WordNet
+[Tamil WordNet](http://www.au-kbc.org/research_areas/nlp/projects/tamil_wordnet.html)
+
+It can be downloaded from:
+[TamilVU](http://www.tamilvu.org/tsdf/downloads/TamilWordNet.zip) or
+[amrita.edu](http://nlp.amrita.edu:8080/project/mhrd/Wn-t.html)
```

Comment:
Second link now seems to be: http://nlp.amrita.edu:8080/project/mhrd/tamil_1.html but the file there is much smaller than the one at tamilvu.org (1.1MB vs 58.6MB).

I downloaded http://www.tamilvu.org/tsdf/downloads/TamilWordNet.zip and it tells me that:

```
Tamil Wordnet is Copyright (c) 2005
by AU-KBC Research Centre, Chennai &
Tamil University, Thanjavur, and
licensed under the terms of 
GNU General Public License (GPL)
```

So unfortunately that seems to mean we can't use this as a data source.